content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -171,55 +171,116 @@ docker run -d --name dd-agent \
 
 {{% tab "Kubernetes" %}}
 
-You can enable APM by installing the Agent with the Datadog Helm chart. This deploys the Datadog Agent across all nodes in your Linux-based Kubernetes cluster with a DaemonSet.
+You can enable APM by installing the Agent with either:
 
-**Note**: Single step instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.
+- [Datadog Operator](#installing-with-datadog-operator)
+- [Datadog Helm chart](#installing-with-helm)
+
+<div class="alert alert-info">Single step instrumentation doesn't instrument applications in the namespace where you install the Datadog Agent. It's recommended to install the Agent in a separate namespace in your cluster where you don't run your applications.</div>
 
 ### Requirements
 
-- Make sure you have [Helm][13] installed.
+- Kubernetes v1.20+
+- [`Helm`][1] for deploying the Datadog Operator.
+- [`Kubectl` CLI][2] for installing the Datadog Agent.
+
+{{< collapse-content title="Installing with Datadog Operator" level="h4" >}}
 
-### Installation
+To enable single step instrumentation with the Datadog Operator:
+
+1. Install the [Datadog Operator][5] with Helm:
+   ```shell
+   helm repo add datadog https://helm.datadoghq.com
+   helm install my-datadog-operator datadog/datadog-operator
+   ```
+2. Create a Kubernetes secret to store your Datadog [API key][3]:
+   ```shell
+   kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY
+   ```
+2. Create `datadog-agent.yaml` with the spec of your Datadog Agent deployment configuration. The simplest configuration is as follows:
+   ```yaml
+   kind: DatadogAgent
+   apiVersion: datadoghq.com/v2alpha1
+   metadata:
+     name: datadog
+   spec:
+     global:
+       site: <DATADOG_SITE>
+       tags:
+         - env:<AGENT_ENV>
+       credentials:
+         apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+         appSecret:
+           secretName: datadog-secret
+           keyName: app-key
+   features:
+     apm:
+       instrumentation:
+         enabled: true  
+   ```
+   Replace `<DATADOG_SITE>` with your [Datadog site][6] and `<AGENT_ENV>` with the environment your Agent is installed on (for example, `env:staging`).
+
+      <div class="alert alert-info">
+      Here you can also optionally configure the following:
+      <ul>
+         <li><a href="#enabling-or-disabling-instrumentation-for-namespaces">Enabling or disabling instrumentation for namespaces.</a></li>
+         <li><a href="#specifying-tracing-library-versions">Specifying tracing library versions.</a></li>
+         <li><a href="/tracing/trace_collection/library_injection_local/">Choosing specific pod specifications.</a></li>
+      </ul>
+   </div>
+
+3. Run the following command:
```

Comment:
After applying datadog-agent.yaml, we should allow few minutes for Datadog Cluster Agent to complete its leader election process. Only after that restarting apps will result in instrumentation.